compiler/Cargo.toml

```diff
@@ -7,13 +7,13 @@ license = "Apache-2.0"
 readme = "README.md"
 
 [dependencies]
-ink_primitives = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_metadata = { git = "https://github.com/paritytech/ink", default-features = false, features = ["derive"], optional = true }
-ink_env = { git = "https://github.com/paritytech/ink", default-features = false, features = ["ink-debug"] }
-ink_storage = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_lang = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_prelude = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_eth_compatibility = { git = "https://github.com/paritytech/ink", default-features = false }
+ink_primitives = { version = "3.0.0-rc9", default-features = false }
+ink_metadata = { version = "3.0.0-rc9", default-features = false, features = ["derive"], optional = true }
+ink_env = { version = "3.0.0-rc9", default-features = false, features = ["ink-debug"] }
+ink_storage = { version = "3.0.0-rc9", default-features = false }
+ink_lang = { version = "3.0.0-rc9", default-features = false }
+ink_prelude = { version = "3.0.0-rc9", default-features = false }
+ink_eth_compatibility = { version = "3.0.0-rc9", default-features = false }
 
 scale = { package = "parity-scale-codec", version = "2", default-features = false, features = ["derive"] }
```

Comment:
```suggestion
scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
```

Reply:
So in this case it wasn't quite as simple as `s/rc8/rc9/g`, so make sure to double check the [sample TOML provided by `cargo-contract`](https://github.com/paritytech/cargo-contract/blob/master/templates/new/_Cargo.toml).

Do you have a CI job to ensure that contracts still build when you make a PR? If not, I'd definitely spend some effort on that

Reply:
Yes, I agree, there should be a CI job. If the `cargo contract build` fails, it will not be possible to build the docker file for the ink! compiler, I'll add a CI workflow which does this.

Reply:
I added a CI job to the github workflow which builds the compiler docker image. Among other cases, this will certainly fail when the ink! contract which is associated with the `Cargo.toml` doesn't compile.